src/clients/SpokePoolClient/SpokePoolClient.ts

```diff
@@ -110,32 +109,9 @@ export class SpokePoolClient extends BaseAbstractClient {
     super(eventSearchConfig);
     this.firstBlockToSearch = eventSearchConfig.fromBlock;
     this.latestBlockSearched = 0;
-    this.queryableEventNames = Object.keys(this._queryableEventNames());
```

Comment:
Notable change – This function can no longer be called from the constructor due to JavaScript inheritance rules. In EVMSpokePoolClient, we’re inside the super() call, but its constructor hasn’t completed execution. Since _queryableEventNames is abstract, the underlying logic isn’t fully instantiated with the necessary information yet.